docs/existing-aws-infrastructure.md

```diff
@@ -0,0 +1,94 @@
+* NAT gateways
+* Elastic IPs assigned to NAT gateways
+
+For the `sigs.k8s.io/cluster-api-provider-aws/cluster/<cluster-name>` tag, a value of "owned" is used by Cluster API itself to indicate resources are owned by a specific cluster, and that the lifecycle of the resource is tied to the lifecycle of the cluster. Cluster API recognizes a value of "shared" to indicate a resource may be shared between multiple clusters, and the lifecycle of said resource is not tied to any particular cluster. In this situation---where AWS infrastructure is _not_ being managed by Cluster API---the value should be something **other** than "owned". Using "shared" is acceptable.
+
+For the `sigs.k8s.io/cluster-api-provider-aws/role` tag, Cluster API uses a variety of values to indicate resources are associated with a particular role. Here are the tag values that should be used:
+
+* VPC, route tables, Internet gateway, and NAT gateways should use the value "common".
+* Public subnets should use the value "public".
+* Private subnets should use the value "private".
+* Elastic IPs (for use by the NAT gateways) should use the value "apiserver".
+
+Additionally, for proper operation of the AWS cloud provider, all subnets where Kubernetes nodes reside should also have the `kubernetes.io/cluster/<cluster-name>` tag present.
+
+## Configuring the AWSCluster Specification
+
+Specifying existing infrastructure for Cluster API to use takes place in the specification for the AWSCluster object. Specifically, you will need to add an entry with the VPC ID and the IDs of all applicable subnets into the `networkSpec` field. Here is an example:
+
+```yaml
+spec:
+  networkSpec:
+    vpc:
+      id: vpc-0425c335226437144
+    subnets:
+      - id: subnet-07758a9bc904d06af
+      - id: subnet-0a3507a5ad2c5c8c3
+      - id: subnet-02ad6429dd0532452
+      - id: subnet-02b300779e9d895cf
+      - id: subnet-03d8f353b289b025f
+      - id: subnet-0a2fe03b0d88fa078
+```
+
+When you use `kubectl apply` to apply the Cluster and AWSCluster specifications to the management cluster, Cluster API will use the specified VPC ID and specified subnet IDs, and will not create a new VPC, new subnets, or other associated resources. It _will_, however, create a new ELB and new security groups.
+
+## Configuring the AWSMachine Specification (Optional)
+
+To distribute EC2 instances across different subnets or different AZs, you can add information to the AWSMachine specification. This is optional; without it, Cluster API will deploy to the first private subnet it discovers.
+
+To specify that an EC2 instance should be placed in a specific subnet, add this to the AWSMachine specification:
+
+```yaml
+spec:
+  subnet:
+    id: subnet-0a3507a5ad2c5c8c3
+```
+
+To tell Cluster API that an EC2 instance should be placed in a particular AZ but allow Cluster API to select which subnet in that AZ can be used, add this to the AWSMachine specification:
+
+```yaml
+spec:
+  availabilityZone: "us-west-2a"
```

Comment:
This should be failureDomain for v1alpha3
```suggestion
  failureDomain: "us-west-2a"
```

Reply:
Fixed via 38ce860.